docs/src/app/docs/get-started/page.md

```diff
@@ -121,223 +117,271 @@ Then run it with:
 uvicorn main:app --port 8080
 ```
 
-If everything is set up correctly, sending a GET request to `http://localhost:8080/` should return a response with the value `Hello World`. Let's integrate some cloud infrastructure using LaunchFlow!
+If everything is set up correctly, you should see `Hello World` when you visit `http://localhost:8080/` in your browser.
```

Comment:
nit: http://localhost:8080 should be a link instead of code

Reply:
done